src/pkg/cli/client/playground_test.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/DefangLabs/defang/src/protos/io/defang/v1/defangv1connect"
+	"github.com/bufbuild/connect-go"
+)
+
+func TestServiceDNS(t *testing.T) {
+	baseUrl := "https://fabric-prod1.defang.dev:443" // should not actually need to exist
+	fabricClient := defangv1connect.NewFabricControllerClient(http.DefaultClient, baseUrl, connect.WithGRPC())
+
+	p := PlaygroundClient{
+		GrpcClient: GrpcClient{client: fabricClient, tenantID: "proj1"},
+	}
+	if p.ServiceDNS("service1") != "proj1-service1" {
+		t.Error("ServiceDNS should return the same string")
+	}
```

Comment:
I think we don't really need to construct the fabricClient, this should be suffice:
```
	p := PlaygroundClient{
		GrpcClient: GrpcClient{tenantID: "proj1"},
	}
	if p.ServiceDNS("service1") != "proj1-service1" {
		t.Error("ServiceDNS should return the same string")
	}
```

Reply:
yeah, now it is, but wasn't before this fix. I created the test before the fix ;)